### ISSUE TITLE
Warm Fall Treatments - when to move in the plants? 

### ISSUE THREAD

**Issue by christophe-rd** [christophe-rd/fuelinex#9]
@lizzieinvancouver @wangxm-forest @DeirdreLoughnan 
Since someone wants to use the biochambers for another experiment, we need to reserve them! I don't know exactly when we will move the plants in the chambers, but I thought they could go in 1-2 weeks before the leaf start their senescence (I will need to do some readings to estimate this). 

Just so we are sure to have them for the beginning of the treatment, how does September 1st to November 30th sound?

**Comment by lizzieinvancouver** [comment]
@christophe-rd  Me and @FrederikBaumgarten were discussing issue #14 and realized we should perhaps time the nutrient boost with the warm fall treatment ... so we should decide when to start warm fall. Options are based on budset or SPAD (and do it species-specific, mirrors the spring design but some species may not have the stage such as SEGI) or pick a calendar date (1 September) for all ... @FrederikBaumgarten likes 1 September (or very early September). I could go either way ... do you have a preference?

If we're equivocal we could pick early September since that will be after most of the Mount Rainier field work!

**Comment by christophe-rd** [comment]
> @christophe-rd Me and @FrederikBaumgarten were discussing issue #14 and realized we should perhaps time the nutrient boost with the warm fall treatment ... so we should decide when to start warm fall. Options are based on budset or SPAD (and do it species-specific, mirrors the spring design but some species may not have the stage such as SEGI) or pick a calendar date (1 September) for all ... @FrederikBaumgarten likes 1 September (or very early September). I could go either way ... do you have a preference?
> 
> If we're equivocal we could pick early September since that will be after most of the Mount Rainier field work!

I think this was the initial plan. There are actually 2 nutrient boost treatments: CoolSpring/CoolFall and WarmSpring/WarmFall, so one will have an extended growing season and the other won't. 

That brings me to something else. I agree that we want to mirror the spring design, but as a reminder, the WarmSpring treatments have stayed at Totem. I believe we don't want to do this for the Fall treatment. We want to move the WarmFall treatments into the chambers, right?

If so, yes, I think early September should be fine. 

Also, what is SPAD?

**Comment by christophe-rd** [comment]
@lizzieinvancouver should I email Alyson to confirm the chambers will be reserved for us?

**Comment by lizzieinvancouver** [comment]
> @lizzieinvancouver should I email Alyson to confirm the chambers will be reserved for us?

Yes! And I think we especially need to finalize with her when we promise to be OUT of the chambers.

**Comment by lizzieinvancouver** [comment]
> That brings me to something else. I agree that we want to mirror the spring design, but as a reminder, the WarmSpring treatments have stayed at Totem. I believe we don't want to do this for the Fall treatment. We want to move the WarmFall treatments into the chambers, right?

Yes! We don't want to flip the fall treatment, just do the treatment as planned. 

> Also, what is SPAD?
It's a way to measure greenness, there should be one around the lab ... I saw @FrederikBaumgarten with it last.
